LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 VOKAL Interactive
+Copyright (c) 2014 Vokal
```

Comment:
I googled some and found [this](http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/) among many other references. I as see it we can leave it undated, have the date of latest revision, or make it a date range. Doesn't seem to matter much.